### PR TITLE
Fixing of warning tests about no pictures selected

### PR DESCRIPTION
**Pull request by heysadboy**
Solves this issue https://github.com/fossasia/phimpme-android/issues/151 .

### CODE REVIEW

File: app/src/main/java/vn/mbm/phimp/me/Upload.java [hunk 1]
```diff
@@ -525,12 +529,19 @@ private int fetchAccentColor() {
         private LayoutInflater mInflater;
 
         public ImageAdapter(Context c) {
+
             mInflater = (LayoutInflater) getActivity().getSystemService(Context.LAYOUT_INFLATER_SERVICE);
             path = imagelist.split("#");
+            if (path.length > 0) {
+                noPhotos.setVisibility(View.GONE);
+                Log.e("Hi", "Removing");
```
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/Upload.java]: Remove the unwanted log or make it logical

File: app/src/main/java/vn/mbm/phimp/me/Upload.java [hunk 2]
```diff
@@ -525,12 +529,19 @@ private int fetchAccentColor() {
...
+            }
 
         }
 
         public int getCount() {
+
```
Comment (anantprsd5) [inline app/src/main/java/vn/mbm/phimp/me/Upload.java]: @heysadboy Please remove unwanted new lines.

Comment (heysadboy) [inline app/src/main/java/vn/mbm/phimp/me/Upload.java]: Got it 👍

**Comment by heysadboy** [comment]
@Abhi2424shek Made the changes 👍

**Pull request merged**